pkg/lightning/lightning_test.go

```diff
@@ -143,8 +143,7 @@ func (s *lightningServerSuite) TestRunServer(c *C) {
 	resp.Body.Close()
 
 	go func() {
-		err := s.lightning.RunServer()
-		c.Assert(err, IsNil)
+		s.lightning.RunServer()
```

Comment:
```suggestion
		_ = s.lightning.RunServer()
```

Or the linter may get unhappy...